.storybook/main.ts

```diff
@@ -27,4 +31,25 @@ module.exports = {
     check: true,
     checkOptions: {},
   },
+  babel: babelConfig,
```

Comment:
See https://storybook.js.org/docs/react/configure/babel#custom-config-file for why I broke the build, and just below that for why this fixes things.